### DISCUSSION TITLE
채널 변경 등 동작관련 인식

### DISCUSSION THREAD

**Discussion by Beantree09** [KimMin-Gwan/ICT-convergence-project-contest-2023#34] [General]
PoseNet모듈을 이용해 캠으로 확인해본 결과 인식하는 부분이  팔꿈치,  어깨,  귀, 눈, 코만 인식이 가능합니다.  손가락은 인식이 불가능한거 같아 손목(손)을 활용해 기능 구현을 해야할 것 같습니다. 얼굴(코)를 기준으로 기능구현을 제작할 예정인데 다른 방법이 있다면 의견 남겨주시면 감사하겠습니다

**Comment by KimMin-Gwan** [comment]
말씀하신데로 포즈넷은 손가락을 판단하지 못합니다. 손가락은 mideapip라는 모듈로 판단합니다.
따라서 모션제어 자체는 손목의 좌표를 코나 어깨 좌표를 기준으로 판단하게 하면됩니다.
